src/statmodels.jl

```diff
@@ -542,6 +542,56 @@ function show(io::IO, ct::CoefTable)
     nothing
 end
 
+show(mime::MIME"text/markdown", ct::CoefTable) = show(Base.stdout, mime, ct)
+
+function show(io::IO, ::MIME"text/markdown", ct::CoefTable)
+    cols = ct.cols; rownms = ct.rownms; colnms = ct.colnms;
+    nc = length(cols)
+    nr = length(cols[1])
+    if length(rownms) == 0
+        rownms = [lpad("[$i]",floor(Integer, log10(nr))+3) for i in 1:nr]
+    end
+    mat = [j == 1 ? NoQuote(rownms[i]) :
+           j-1 == ct.pvalcol ? NoQuote(sprint(show, PValue(cols[j-1][i]))) :
+           j-1 in ct.teststatcol ? TestStat(cols[j-1][i]) :
```

Comment:
Why is `NoQuote(sprint(...))` needed for `PValue` but not for `TestStat`?

Reply:
Because of the `<` when printing small `PValue`.